release_notes/v2_vs_v3.adoc

```diff
@@ -16,6 +16,10 @@ Many of the same terms are used, and the same functions are performed, but the t
 
 This topic discusses these differences in detail, in an effort to help OpenShift users in the transition from OpenShift v2 to OpenShift v3.
 
+ifdef::openshift-dedicated[]
+OpenShift Dedicated uses the same code base as OpenShift Enterprise 3. OpenShift Dedicated is hosted on the public cloud and managed by Red Hat. An OpenShift Enterprise 3 subscription entitles you to host and manage the software on your infrastructure of choice.
+endif::[]
```

Comment:
On its own (without juxtaposing the "v#" language), I'm not sure that "OpenShift Dedicated 3 follows the numbering of the product's major version" is worth saying. At that point, I'd probably just drop it and leave the bit about the code base.